src/Store/reducers/sharedSiteInfoReducer.js

```diff
@@ -10,9 +10,9 @@ export const sharedSiteInfoReducer = (state = defaultState, action = {}) => {
     case 'SET_VALUES_EDITED':
       return setValuesEdited(state, action);
     case 'SET_EDIT_MODAL_OPEN':
-      return setEditModalOpen(state, action);
+      return setEditLocationModalOpen(state, action);
     case 'SET_EDIT_MODAL_DATA':
```

Comment:
will you change the values in the switch to match the new action names? ie SET_EDIT_MODAL_DATA -> SET_EDIT_LOCATION_MODAL_DATA etc.